ci/jenkins/test-rancher.sh

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+DEFAULT_WORKDIR=$HOME
+WORKDIR=$DEFAULT_WORKDIR
+KUBECONFIG_PATH=""
+TESTCASE=""
+TEST_FAILURE=false
+MODE="report"
+DOCKER_REGISTRY=$(head -n1 "${WORKSPACE}/ci/docker-registry")
+GO_VERSION=$(head -n1 "${WORKSPACE}/build/images/deps/go-version")
+IMAGE_PULL_POLICY="Always"
+
+CONFORMANCE_SKIP="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[sig-cli\]|\[sig-storage\]|\[sig-auth\]|\[sig-api-machinery\]|\[sig-apps\]|\[sig-node\]|\[sig-instrumentation\]"
+NETWORKPOLICY_SKIP=""
```

Comment:
Is there any reason to use custom _SKIP variables here? It's better to use the default _SKIP variable values of ci/run-k8s-e2e-tests.sh. If this script doesn't pass --e2e-skip arguments to ci/run-k8s-e2e-tests.sh, ci/run-k8s-e2e-tests.sh just uses the following values. Maybe we don't need to define them and pass them again in this script?

```
ci/run-k8s-e2e-tests.sh
DEFAULT_E2E_CONFORMANCE_SKIP="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[sig-cli\]|\[sig-storage\]|\[sig-auth\]|\[sig-api-machinery\]|\[sig-apps\]|\[sig-node\]|\[sig-instrumentation\]"
DEFAULT_E2E_NETWORKPOLICY_SKIP=""
```